src/libaktualizr/package_manager/packagemanagerfactory_test.cc

```diff
@@ -11,41 +11,32 @@
 
 boost::filesystem::path sysroot;
 
-/* Support OSTree as a package manager. */
+#ifdef BUILD_OSTREE
 TEST(PackageManagerFactory, Ostree) {
   Config config;
   config.pacman.type = PACKAGE_MANAGER_OSTREE;
   config.pacman.sysroot = sysroot;
   TemporaryDirectory dir;
   config.storage.path = dir.Path();
   std::shared_ptr<INvStorage> storage = INvStorage::newStorage(config.storage);
-#ifdef BUILD_OSTREE
   std::shared_ptr<PackageManagerInterface> pacman =
       PackageManagerFactory::makePackageManager(config.pacman, config.bootloader, storage, nullptr);
   EXPECT_TRUE(pacman);
-#else
-  EXPECT_THROW(std::shared_ptr<PackageManagerInterface> pacman =
-                   PackageManagerFactory::makePackageManager(config.pacman, config.bootloader, storage, nullptr),
-               std::runtime_error);
 #endif
 }
```

Comment:
`#endif` probably has to come after the closing brace.

Reply:
! thanks